### PR TITLE
feature/drawer menu

### PR DESCRIPTION
**Pull request by josanri**
Renewed menu from burger at the same position to drawer at right (tablets) or top (mobile). Also changed icons for dark and light mode and aligned to vertical center

### CODE REVIEW

**Review by davidrojom — CHANGES_REQUESTED** [review]
Dark mode button disappears above 1280px viewport
image

File: src/components/Navigation.astro [hunk 1]
```diff
@@ -10,9 +10,11 @@ const featuredSponsors = DATA.sponsors.filter(
 
 <nav
   id="top"
-  class="flex flex-wrap items-center justify-between w-full mx-auto max-w-[2000px] px-8 py-5 md:px-16 md:py-10 text-lg text-dark-background dark:text-light-background"
+  class="flex flex-wrap
+  items-center justify-between w-full mx-auto max-w-[2000px] px-8 py-5 md:px-16 md:py-10 text-lg text-dark-background dark:text-light-background
+  bg-white dark:bg-dark-background"
 >
-  <div class="w-full max-w-[150px] sm:max-w-[180px] md:max-w-[200px]">
+  <div class="w-full max-w-[150px] sm:max-w-[180px] md:max-w-[200px] max-xl:z-50">
```
Comment (davidrojom) [inline src/components/Navigation.astro:17]: I think you want to user xl:z-50

File: src/components/Navigation.astro [hunk 2]
```diff
@@ -21,26 +23,33 @@ const featuredSponsors = DATA.sponsors.filter(
...
+    transition-transform max-xl:translate-x-full
+    max-xl:bg-white max-xl:dark:bg-dark-background
+    max-xl:duration-0"
+    tabindex="-1"
   >
-    <ul
-      class="flex justify-evenly gap-2 text-sm flex-col lg:flex-row font-bold"
+  <ul
+      class="flex justify-evenly
+      xl:gap-2 max-xl:gap-10
```
Comment (davidrojom) [inline src/components/Navigation.astro]: I think you want to use gap-2 xl:gap-10
image

File: src/components/Navigation.astro [hunk 3]
```diff
@@ -21,26 +23,33 @@ const featuredSponsors = DATA.sponsors.filter(
...
+      max-xl:mt-28
+      text-sm max-xl:flex-col xl:flex-row font-bold max-xl:gap-8"
```
Comment (davidrojom) [inline src/components/Navigation.astro:46]: I think you want to use xl:gap-8 instead

**Review by davidrojom — CHANGES_REQUESTED** [review]

**Comment by davidrojom** [comment]
Glitched now when resizing the viewport

Recording-2024-09-21-160204

**Comment by DavidRojoM** [comment]
Working as a charm.

What do you think of changing drawer's max-w to 300px instead of 700?

> 700px
image

> 300px
image

**Comment by josanri** [comment]
Have to change it to be done only at max-xl, but would mobile is still full width or not?
https://github.com/biznaga-fest/web/pull/9#issuecomment-2365209810

**Comment by davidrojom** [comment]
nevermind it, merging. Thanks for the contributions, keep up the good work!

**Pull request merged**